openrouteservice-api-tests/src/test/java/org/heigit/ors/v2/services/isochrones/fast/ResultTest.java

```diff
@@ -380,7 +380,7 @@ public void testSmoothingFactor() {
                 .then()
                 .body("any { it.key == 'type' }", is(true))
                 .body("any { it.key == 'features' }", is(true))
-                .body("features[0].geometry.coordinates[0].size", is(69))
+                .body("features[0].geometry.coordinates[0].size()", is(69))
```

Comment:
Could you maybe briefly clarify, how this change relates to the PR?

Reply:
not at all, but the notation without the parentheses causes problems when run on newer version JVMs